Index: node_modules/fullcalendar/dist/fullcalendar.js
===================================================================
--- softwerkskammer/build/javascript/fullcalendar.js	(revision )
+++ softwerkskammer/build/javascript/fullcalendar.js	(revision )
@@ -5043,13 +5043,8 @@
 
 		classes.unshift('fc-day-grid-event', 'fc-h-event');
 
-		// Only display a timed events time if it is the starting segment
-		if (seg.isStart) {
-			timeText = this.getEventTimeText(event);
-			if (timeText) {
-				timeHtml = '<span class="fc-time">' + htmlEscape(timeText) + '</span>';
-			}
-		}
+		// Never display a timed events time if it is the starting segment
+		// Original code deleted here
 
 		titleHtml =
 			'<span class="fc-title">' +
@@ -8678,7 +8673,6 @@
 				var groupEl;
 
 				$.each(this.split(','), function(j, buttonName) {
-					var viewSpec;
 					var buttonClick;
 					var overrideText; // text explicitly set by calendar's constructor options. overcomes icons
 					var defaultText;
@@ -8717,11 +8711,14 @@
 							normalIcon = options.buttonIcons[buttonName];
 
 							if (overrideText) {
-								innerHtml = htmlEscape(overrideText);
+								innerHtml = overrideText;
 							}
 							else if (themeIcon && options.theme) {
 								innerHtml = "<span class='ui-icon ui-icon-" + themeIcon + "'></span>";
 							}
+							else if (normalIcon && options.bootstrap) {
+								innerHtml = "<i class='fa " + normalIcon + "'></i>";
+							}
 							else if (normalIcon && !options.theme) {
 								innerHtml = "<span class='fc-icon fc-icon-" + normalIcon + "'></span>";
 							}
@@ -8729,17 +8726,25 @@
 								innerHtml = htmlEscape(defaultText);
 							}
 
+							if (options.bootstrap) {
+								button = $(
+									'<span class="fc-' +
+									buttonName + '-button btn btn-default">' +
+									innerHtml + '</span>'
+								)
+							} else {
-							classes = [
-								'fc-' + buttonName + '-button',
-								tm + '-button',
-								tm + '-state-default'
-							];
+								classes = [
+									'fc-' + buttonName + '-button',
+									tm + '-button',
+									tm + '-state-default'
+								];
-
-							button = $( // type="button" so that it doesn't submit a form
-								'<button type="button" class="' + classes.join(' ') + '">' +
-									innerHtml +
-								'</button>'
+								button = $( // type="button" so that it doesn't submit a form
+									'<button type="button" class="' + classes.join(' ') + '">' +
+										innerHtml +
+									'</button>'
 								)
+							}
+							button
 								.click(function() {
 									// don't process clicks for disabled buttons
 									if (!button.hasClass(tm + '-state-disabled')) {
@@ -8790,7 +8795,7 @@
 					}
 				});
 
-				if (isOnlyButtons) {
+				if (isOnlyButtons && !options.bootstrap) {
 					groupChildren
 						.first().addClass(tm + '-corner-left').end()
 						.last().addClass(tm + '-corner-right').end();
@@ -8799,7 +8804,11 @@
 				if (groupChildren.length > 1) {
 					groupEl = $('<div/>');
 					if (isOnlyButtons) {
+						if (options.bootstrap) {
+							groupEl.addClass('btn-group');
+						} else {
-						groupEl.addClass('fc-button-group');
+							groupEl.addClass('fc-button-group');
+						}
 					}
 					groupEl.append(groupChildren);
 					sectionEl.append(groupEl);
