Index: node_modules/fullcalendar/dist/fullcalendar.js
===================================================================
--- softwerkskammer/build/javascript/fullcalendar.js	(revision )
+++ softwerkskammer/build/javascript/fullcalendar.js	(revision )
@@ -6934,12 +6934,7 @@
 		classes.unshift('fc-day-grid-event', 'fc-h-event');
 
 		// Only display a timed events time if it is the starting segment
-		if (seg.isStart) {
-			timeText = this.getEventTimeText(event);
-			if (timeText) {
-				timeHtml = '<span class="fc-time">' + htmlEscape(timeText) + '</span>';
-			}
-		}
+		// Original code deleted here
 
 		titleHtml =
 			'<span class="fc-title">' +
@@ -10725,89 +10720,104 @@
 									calendar.options.buttonIcons[buttonName];
 
 							if (overrideText) {
-								innerHtml = htmlEscape(overrideText);
+								innerHtml = overrideText;
 							}
 							else if (themeIcon && calendar.options.theme) {
 								innerHtml = "<span class='ui-icon ui-icon-" + themeIcon + "'></span>";
 							}
+							else if (normalIcon && calendar.options.bootstrap) {
+								innerHtml = "<i class='fa " + normalIcon + "'></i>";
+							}
 							else if (normalIcon && !calendar.options.theme) {
 								innerHtml = "<span class='fc-icon fc-icon-" + normalIcon + "'></span>";
 							}
 							else {
 								innerHtml = htmlEscape(defaultText);
 							}
 
+							if (calendar.options.bootstrap) {
+								button = $(
+									'<span class="fc-' +
+									buttonName + '-button btn btn-default">' +
+									innerHtml + '</span>'
+								)
+							} else {
-							classes = [
-								'fc-' + buttonName + '-button',
-								tm + '-button',
-								tm + '-state-default'
-							];
+								classes = [
+									'fc-' + buttonName + '-button',
+									tm + '-button',
+									tm + '-state-default'
+								];
-
-							button = $( // type="button" so that it doesn't submit a form
-								'<button type="button" class="' + classes.join(' ') + '">' +
+								button = $( // type="button" so that it doesn't submit a form
+									'<button type="button" class="' + classes.join(' ') + '">' +
 									innerHtml +
-								'</button>'
+									'</button>'
 								)
-								.click(function(ev) {
+							}
+							button
+								.click(function() {
 									// don't process clicks for disabled buttons
 									if (!button.hasClass(tm + '-state-disabled')) {
 
-										buttonClick(ev);
+										buttonClick();
 
 										// after the click action, if the button becomes the "active" tab, or disabled,
 										// it should never have a hover class, so remove it now.
										if (
											button.hasClass(tm + '-state-active') ||
											button.hasClass(tm + '-state-disabled')
										) {
											button.removeClass(tm + '-state-hover');
										}
									}
								})
								.mousedown(function() {
									// the *down* effect (mouse pressed in).
									// only on buttons that are not the "active" tab, or disabled
									button
										.not('.' + tm + '-state-active')
										.not('.' + tm + '-state-disabled')
										.addClass(tm + '-state-down');
								})
								.mouseup(function() {
									// undo the *down* effect
 									button.removeClass(tm + '-state-down');
 								})
 								.hover(
-									function() {
-										// the *hover* effect.
-										// only on buttons that are not the "active" tab, or disabled
-										button
-											.not('.' + tm + '-state-active')
-											.not('.' + tm + '-state-disabled')
-											.addClass(tm + '-state-hover');
-									},
-									function() {
-										// undo the *hover* effect
-										button
-											.removeClass(tm + '-state-hover')
-											.removeClass(tm + '-state-down'); // if mouseleave happens before mouseup
-									}
-								);
+								function() {
+									// the *hover* effect.
+									// only on buttons that are not the "active" tab, or disabled
+									button
+										.not('.' + tm + '-state-active')
+										.not('.' + tm + '-state-disabled')
+										.addClass(tm + '-state-hover');
+								},
+								function() {
+									// undo the *hover* effect
+									button
+										.removeClass(tm + '-state-hover')
+										.removeClass(tm + '-state-down'); // if mouseleave happens before mouseup
+								}
+							);
 
 							groupChildren = groupChildren.add(button);
 						}
 					}
 				});
 
-				if (isOnlyButtons) {
+				if (isOnlyButtons && !calendar.options.bootstrap) {
 					groupChildren
 						.first().addClass(tm + '-corner-left').end()
 						.last().addClass(tm + '-corner-right').end();
				}

 				if (groupChildren.length > 1) {
 					groupEl = $('<div/>');
 					if (isOnlyButtons) {
+						if (calendar.options.bootstrap) {
+							groupEl.addClass('btn-group');
+						} else {
-						groupEl.addClass('fc-button-group');
+							groupEl.addClass('fc-button-group');
+						}
 					}
 					groupEl.append(groupChildren);
 					sectionEl.append(groupEl);
